proposals/0167-app-services.md

```diff
@@ -471,38 +471,38 @@ The `AppServiceManifest` is essentially detailing everything about a particular
 
 ```xml
 	<struct name="AppServiceManifest">
-		<description> This manfifest contains all the information necessary for the service to be published, activated, and consumers able to interact with it</description>
+		<description> This manifest contains all the information necessary for the service to be published, activated, and consumers able to interact with it</description>
```

Comment:
Update to:
		<description> This manifest contains all the information necessary for the service to be published, activated, **and allow consumers to interact with it**</description>